### PR TITLE
Fix Up Delinquent Changelogs

### PR DESCRIPTION
**Pull request by chidozieononiwu**

### CODE REVIEW

File: sdk/storagesync/Microsoft.Azure.Management.StorageSync/CHANGELOG.md [hunk 1]
```diff
@@ -1,26 +1,2 @@
-## Microsoft.Azure.Management.StorageSync release notes
+# Release History
 
-### Changes in 2020-03-01
```
Comment (weshaggard) [inline sdk/storagesync/Microsoft.Azure.Management.StorageSync/CHANGELOG.md:3]: This is a case where there are no matching headers and we end up throwing out everything. We should figure out about saving this in some sort of empty placeholder.

File: sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md [hunk 2]
```diff
@@ -79,20 +96,3 @@ Note that these breaking changes are only breaking changes from the **preview**
 - [Azure Digital Twins Public Repo](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/digitaltwins/Azure.DigitalTwins.Core)
 - [Azure Digital Twins Samples](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/digitaltwins/Azure.DigitalTwins.Core/samples)
 
-## 1.0.0-preview.1 (Unreleased)
```
Comment (weshaggard) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md:82]: This looks like a case where the date never got stamped in.

File: sdk/cognitiveservices/Knowledge.QnAMaker/CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +14,3 @@
   For Minor Version release
 ## 2.0.0 (2020-07-16)
   Remove preview tag from the release version
-## 2.0.0-preview.1 (Unreleased)
```
Comment (chidozieononiwu) [inline sdk/cognitiveservices/Knowledge.QnAMaker/CHANGELOG.md:12]: @bingisbestest @nerajput1607 Can we remove this entry?

File: sdk/storagesync/Microsoft.Azure.Management.StorageSync/CHANGELOG.md (hunk 1 above)
Comment (chidozieononiwu) [reply sdk/storagesync/Microsoft.Azure.Management.StorageSync/CHANGELOG.md]: Its management plane so I left it like that.

File: sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md (hunk 2 above)
Comment (chidozieononiwu) [reply sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md:88]: It was never released so I removed it.

**Review by chamons — COMMENTED** [review]
Synapse change is just newline, so no complaints from me.

**Review by heaths — APPROVED** [review]
Track 1 Key Vault changes LGTM

**Pull request merged**